From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 21 Jul 2020 13:11:18 +0000
Subject: Use legacy rollup-plugin-buble

In this version, the import changed to @rollup/plugin-buble (a path not
yet available in debian). This patch temporarily restores the old import
path.
---
 rollup.config.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rollup.config.js b/rollup.config.js
index f50f62f..4661757 100644
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,4 +1,4 @@
-import buble from '@rollup/plugin-buble';
+import buble from 'rollup-plugin-buble';
 
 export default [
   {
